aws/network-primitives/scripts/create_bootstrap_node_compose_file.sh

```diff
@@ -51,7 +51,8 @@ services:
     environment:
       - RUST_LOG=info
     ports:
-      - "30533:30533"
+      - "30533:30533/tcp"
+      - "30533:30533/udp"
```

Comment:
UDP should come first as higher priority as mentioned in Slack for consistency with priority